packages/react-shared-contexts/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@fluentui/react-shared-contexts",
+  "version": "1.0.0-beta.0",
+  "description": "Fluent UI React Contexts shared by multiple components.",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "typings": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "lint": "just-scripts lint",
+    "just": "just-scripts",
+    "code-style": "just-scripts code-style",
+    "clean": "just-scripts clean",
+    "start-test": "just-scripts jest-watch",
+    "update-api": "just-scripts update-api",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.0",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/jest": "~24.9.0",
+    "@types/node": "^10.3.2",
+    "@types/prop-types": "15.7.1",
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/sinon": "2.2.2",
+    "@fluentui/scripts": "^1.0.0",
+    "@fluentui/jest-serializer-merge-styles": "^8.0.0-beta.1",
+    "@fluentui/test-utilities": "^8.0.0-beta.1",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0",
+    "sinon": "^4.1.3"
+  },
+  "dependencies": {
+    "@fluentui/set-version": "^8.0.0-beta.0",
+    "prop-types": "^15.7.2",
+    "tslib": "^1.10.0"
+  },
```

Did you generate this with `yarn create-package` or by copying something? Several of these deps shouldn't be here. (I'm fine with either removing the test-related deps, or keeping them if you think there might be tests later.)
```suggestion
  "devDependencies": {
    "@fluentui/eslint-plugin": "^1.0.0-beta.0",
    "@types/enzyme": "3.10.3",
    "@types/enzyme-adapter-react-16": "1.0.3",
    "@types/jest": "~24.9.0",
    "@types/react": "16.8.25",
    "@types/react-test-renderer": "^16.0.0",
    "@fluentui/scripts": "^1.0.0",
    "@fluentui/test-utilities": "^8.0.0-beta.1",
    "enzyme": "~3.10.0",
    "enzyme-adapter-react-16": "^1.15.0",
    "react": "16.8.6",
    "react-test-renderer": "^16.3.0"
  },
  "dependencies": {
    "@fluentui/set-version": "^8.0.0-beta.0",
    "tslib": "^1.10.0"
  },
```

---

Alternate version if you want to remove the test stuff (in that case you should also remove the jest config and test-related scripts)
```suggestion
  "devDependencies": {
    "@fluentui/eslint-plugin": "^1.0.0-beta.0",
    "@types/react": "16.8.25",
    "@fluentui/scripts": "^1.0.0",
    "react": "16.8.6"
  },
  "dependencies": {
    "@fluentui/set-version": "^8.0.0-beta.0",
    "tslib": "^1.10.0"
  },
```

---

There might be tests later, so I'd prefer to keep them there. This is by copying since the `create-package` script failed several times for me. I plan to look into it later.

---

Done!